gctscript/modules/gct/gct_types.go

```diff
@@ -13,8 +13,10 @@ const (
 	ErrEmptyParameter = "received empty parameter for %v"
 )
 
-var errInvalidInterval = errors.New("invalid interval")
-var supportedDurations = []string{"1m", "3m", "5m", "15m", "30m", "1h", "2h", "4h", "6h", "12h", "24h", "1d", "3d", "1w"}
+var (
+	errInvalidInterval = errors.New("invalid interval")
+	supportedDurations = []string{"1m", "3m", "5m", "15m", "30m", "1h", "2h", "4h", "6h", "12h", "24h", "1d", "3d", "1w"}
```

Comment:
OSS: missing 1M and this should really be checking against exchange capabilities.